README.md

```diff
@@ -72,4 +72,13 @@ Run the Tests
 poetry run flask run
 
 Changes to template
-index.html
\ No newline at end of file
+index.html
+
+Module-3-Automated Testing
```

Comment:
We should provide some advice here on how to run the tests, e.g. with `poetry run pytest`, and any dependencies (like Chromedriver executable, and Chrome)